homeassistant/components/influxdb/__init__.py

```diff
@@ -134,11 +135,15 @@ def setup(hass, config):
         influx.write_points([])
     except (exceptions.InfluxDBClientError,
             requests.exceptions.ConnectionError) as exc:
-        _LOGGER.error("Database host is not accessible due to '%s', please "
-                      "check your entries in the configuration file (host, "
-                      "port, etc.) and verify that the database exists and is "
-                      "READ/WRITE", exc)
-        return False
+        _LOGGER.warning(
+            "Database host is not accessible due to '%s', please "
+            "check your entries in the configuration file (host, "
+            "port, etc.) and verify that the database exists and is "
+            "READ/WRITE. Retrying again in %s sec.", exc, RETRY_INTERVAL
+        )
+        event_helper.call_later(
+            hass, RETRY_INTERVAL, lambda _: setup(hass, config)
+        )
```

Comment:
You should `return True` here or the code below will create a new thread for each retry.

Reply:
Fixed it by my latest commit